sirepo/package_data/static/js/srw.js

```diff
@@ -2656,8 +2656,8 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                     j.jobRunMode = 'parallel';
                 }
                 frameCache.setFrameCount(0, $scope.model);
+                appState.saveQuietly($scope.simState.model);
```

Comment:
Why does this work, and how does it affect other models. What does the quite mode modification do?

Reply:
Gurhar and I discussed this. The bug was the changes to the simState.model were not being saved except for the multiElectronAnimation case. The saveQuietly saves the data with the expectation that another save (or autosave) will push it to the server. There is already a saveAndRun() call in this method, so no need to call saveChanges() followed by saveAndRunSimulation() because it will post to the server twice.